src/LodeRunner.API.Test/IntegrationTests/Controllers/TestRuns.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using LodeRunner.Core.Models;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace LodeRunner.API.Test.IntegrationTests.Controllers
+{
+    /// <summary>
+    /// Represents TestRuns.
+    /// </summary>
+    public class TestRuns : IClassFixture<ApiWebApplicationFactory<Startup>>
+    {
+        private const string TestRunsUri = "/api/TestRuns";
+
+        private readonly ApiWebApplicationFactory<Startup> factory;
+
+        private readonly JsonSerializerOptions jsonOptions;
+
+        private readonly ITestOutputHelper output;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TestRuns"/> class.
+        /// </summary>
+        /// <param name="factory">The factory.</param>
+        /// <param name="output">The output.</param>
+        public TestRuns(ApiWebApplicationFactory<Startup> factory, ITestOutputHelper output)
+        {
+            this.factory = factory;
+
+            this.output = output;
+
+            this.jsonOptions = new ()
+            {
+                IgnoreNullValues = true,
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                DictionaryKeyPolicy = JsonNamingPolicy.CamelCase,
+                PropertyNameCaseInsensitive = true,
+            };
+            this.jsonOptions.Converters.Add(new JsonStringEnumConverter());
+        }
+
+        /// <summary>
+        /// Determines whether this instance [can get test runs].
+        /// </summary>
+        /// <returns><see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        [Trait("Category", "Integration")]
+        public async Task CanGetTestRuns()
+        {
+            using var httpClient = ComponentsFactory.CreateLodeRunnerAPIHttpClient(this.factory);
+
+            HttpResponseMessage httpResponse = await httpClient.GetTestRuns(TestRunsUri, this.output);
+            Assert.Contains(httpResponse.StatusCode, new List<HttpStatusCode> { HttpStatusCode.OK, HttpStatusCode.NoContent });
+

```

Comment:
Should this return content or not?  If it is supposed to return content then how does it get populated ahead of attempting to get them as I didn't see it in there?

Reply:
After discussion, it was to accommodate an unknown state due to the order in testing.  We'll keep it like this for the moment as we have a plan to move away from this testing and drive the API externally.